apps/src/applab/PlaySpaceHeaderStyles.js

```diff
@@ -0,0 +1,47 @@
+/** @file Reusable styles for components used by PlaySpaceHeader. */
+// Strict linting: Absorb into global config when possible
+/* jshint
+ unused: true,
+ eqeqeq: true,
+ maxlen: 120
+ */
+
+module.exports = {
+  buttonStyle: {
+    display: 'inline-block',
+    verticalAlign: 'top',
+    border: '1px solid #949ca2',
+    margin: '0 0 8px 0',
+    padding: '2px 6px',
+    fontSize: 14
+  },
+  toggleButtonStyle: {
+    borderRightWidth: '0 !important',
```

Comment:
Making this rule `!important` fixes the one-pixel jump when clicking the "Code" button, which is caused by a 1px !important top-level border style on buttons in common.css.